assets/js/blocks/product-gallery/inner-blocks/product-gallery-large-image-next-previous/block.json

```diff
@@ -9,5 +9,16 @@
     "keywords": [ "WooCommerce" ],
 	"usesContext": [ "nextPreviousButtonsPosition", "productGalleryClientId" ],
     "textdomain": "woo-gutenberg-products-block",
+	"supports": {
+		"layout": {
```

Comment:
For adding the support to WordPress 6.2, I think that it is necessary do this change:
```suggestion
		"__experimentalLayout": {
```

Reply:
Do you think we need to support WordPress 6.2. with this?

Seems like WooCommerce Blocks currently requires at least WP 6.3.

Reply:
I have tested this with WP `6.2.2` and `__experimentalLayout` and noticed that it's not working correctly out of the box:
![Ks0GBAQrVu](https://github.com/woocommerce/woocommerce-blocks/assets/905781/cd2e6061-d7a4-4d35-b283-b5dc21399978)